### PR TITLE
Delete mpress.rb

### PR DESCRIPTION
**Pull request by miccal**
@MikeMcQuaid this formula currently does not work on my machine (macOS 10.12.2):
it installs fine, but when I run `brew test mpress` the output is:
```
-bash-4.4.5- /Users/raesy [> brew test mpress
Testing homebrew/binary/mpress
==> Using the sandbox
==> /usr/local/Cellar/mpress/1.21_1/bin/mpress
Error: homebrew/binary/mpress: failed
<1> expected but was
<22>.
-bash-4.4.5- /Users/raesy [>
```
and running `mpress --version` outputs nothing.

Downloading the file manually and running the binary directly via `mmpress --version` outputs `Killed: 9`, as noted [here](https://github.com/Homebrew/homebrew-binary/pull/252).

The suggestion in the link above was to split it into separate i386 and x86_64 binaries, as only the x86_64 binary has a problem.

Running the command `lipo -thin i386 mmpress -o mpress1` as suggested in the formula creates a binary that fails, but with no output, just as with installing the formula via `brew install mpress`.

Finally, running the command `lipo -thin x86_64 mmpress -o mpress2` creates a binary that has the same `Killed: 9` output, confirming the problem is indeed with the x86_64 portion.

In summary, I am guessing that the i386 version does not run on the latest macOS.

So, I would suggest this one just gets deleted (unless you have any other thoughts?).

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
Agreed to just deleting it. If someone can get it working it can go to Cask later. Thanks!
